python/ray/dag/compiled_dag_node.py

```diff
@@ -214,7 +217,7 @@ def __init__(self, idx: int, dag_node: "ray.dag.DAGNode"):
         self.dag_node = dag_node
 
         self.downstream_node_idxs: Dict[int, "ray.actor.ActorHandle"] = {}
-        self.output_channel = None
+        self.output_channel: Optional[ChannelInterface] = None
```

Comment:
Hmm I realized from #47098 that probably it will actually be simpler to allocate a separate output channel per task return. Can you update the code to do this?

Reply:
Ack.